rust/operator-binary/src/druid_controller.rs

```diff
@@ -550,14 +576,15 @@ fn build_rolegroup_statefulset(
         .get(&rolegroup_ref.role_group);
 
     // init container builder
-    let mut cb = ContainerBuilder::new(APP_NAME);
+    let mut cb = ContainerBuilder::new(APP_NAME).expect("ContainerBuilder not created");
```

Comment:
Currently failing here is fine since we provide it ourselves. 
This would be the validation https://github.com/stackabletech/operator-rs/blob/c96d6babaf8d417bf9f08041fd0bc36102ea6a82/src/builder/pod/container.rs#L235.
I still would rather add an error here to not forget it in the future.

Reply:
Removed.

Reply:
Yeah thats what i meant :)